lib/std/math.zig

```diff
@@ -1122,6 +1122,28 @@ pub fn isPowerOfTwo(v: anytype) bool {
     return (v & (v - 1)) == 0;
 }
 
+/// Extends the given integer type bit width to a width divisible by 8.
```

Comment:
Why not `ByteAlignedInt` ? ExtendedInt is problematic, because we might want to rely on it for bigint compile_rt routines due to prior art (llvm utilizing it). See https://github.com/ziglang/zig/issues/15682.

Unless you can think of a better name for the bigint compiler_rt ones.

Reply:
Good point. I'll rename it later